### ISSUE TITLE
Unable to use equals sign (=) in command line argument values

### ISSUE THREAD

**Issue by lazyboy1** [danielpalme/ReportGenerator#401]
**Describe the bug**

Report Generator throws exception when a command line argument contains equals sign (=) in value. 

**To Reproduce**

Run this command for example:
```
reportgenerator -reports:**/coverage*.xml -targetdir:coverage-report "-reporttypes:JsonSummary;Html" -verbosity:Verbose "-title:my=title"
```

Output:
```
2020-12-22T14:50:32: Arguments
2020-12-22T14:50:32:  -reports:**/coverage*.xml
2020-12-22T14:50:32:  -targetdir:coverage-report
2020-12-22T14:50:32:  -reporttypes:JsonSummary;Html
2020-12-22T14:50:32:  -verbosity:Verbose
2020-12-22T14:50:32:  -title:my=title
2020-12-22T14:50:32: The short switch '-title:my=title' is not defined in the switch mappings.
2020-12-22T14:50:32:    at Microsoft.Extensions.Configuration.CommandLine.CommandLineConfigurationProvider.Load()
   at Microsoft.Extensions.Configuration.ConfigurationRoot..ctor(IList`1 providers)
   at Microsoft.Extensions.Configuration.ConfigurationBuilder.Build()
   at Palmmedia.ReportGenerator.Core.Generator.GetConfiguration() in /_/src/ReportGenerator.Core/Generator.cs:line 275
   at Palmmedia.ReportGenerator.Core.Generator.GenerateReport(IReportConfiguration reportConfiguration) in /_/src/ReportGenerator.Core/Generator.cs:line 44
```

**Comment by danielpalme** [comment]
Fixed in version 4.8.3

**Comment by lazyboy1** [comment]
Thanks for the quick fix!
